src/libmusig2.c

```diff
@@ -358,14 +366,19 @@ int musig2_aggregate_partial_sig(secp256k1_context *ctx, musig2_context *mca, mu
     secp256k1_xonly_pubkey xonly_R; // x_only R
 
     mca->ctx = secp256k1_context_clone(ctx);
+    for (i = 0; i < nr_signers; i++) {
+        if (mps[i].sig[0] == '\0'){
+            return -2 ;
+        }
```

Comment:
This is probably the reason why things fail in the CI. Try performing this check in a better way.